### PR TITLE
Posthog improvements

### PR DESCRIPTION
**Pull request by hughns**
Adds cryptoVersion to platform details.
Adds CallEnded.roomEventEncryptionKeysSent.
Adds CallEnded.roomEventEncryptionKeysReceived.
Adds CallEnded.roomEventEncryptionKeysReceivedAverageAge.

### CODE REVIEW

File: src/room/GroupCallView.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const GroupCallView: FC<Props> = ({
   const { displayName, avatarUrl } = useProfile(client);
   const roomName = useRoomName(rtcSession.room);
   const roomAvatar = useRoomAvatar(rtcSession.room);
-  const { perParticipantE2EE, returnToLobby } = useUrlParams();
```
Comment (robintown) [inline src/room/GroupCallView.tsx]: Does this mean we no longer respect the perParticipantE2EE URL parameter? Can it be removed?

**Review by robintown — COMMENTED** [review]
One question:

File: src/room/GroupCallView.tsx (hunk 1 above)
Comment (hughns) [reply src/room/GroupCallView.tsx]: This isn't the intention. I will have a look and check that this isn't the case.

**Comment by hughns** [comment]
The js-sdk bump landed in https://github.com/element-hq/element-call/pull/2638.

File: src/room/GroupCallView.tsx (hunk 1 above)
Comment (hughns) [reply src/room/GroupCallView.tsx]: I've removed this part of the PR for now.

**Review by AndrewFerr — APPROVED** [review]

**Pull request merged**